SpatialGDK/Documentation/known-issues.md

```diff
@@ -23,5 +23,5 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | NetMulticast RPCs do not work on Singleton Actors. | 2019-03-13 | [UNR-856](https://improbableio.atlassian.net/browse/UNR-856) | Do not use NetMulticast RPCs on Singleton Actors.|
 | Replicated TimelineComponents are not supported. | 2019-03-14 | [UNR-1105](https://improbableio.atlassian.net/browse/UNR-1105) | Use an alternative method of achieving the desired behaviour, such as playing the timeline on a multicast event, or replicating what is changed by the timeline. |
 | Generating schema for `UClass`es that are not explicitly loaded/referenced by the currently loaded level is not supported. For example, a hard coded path loaded at run-time will not be detected by the schema generator. | 2019-03-25 |  | Reference the class explicitly from an already loaded class (Level blueprint, GameMode etc). Alternatively, you can open the Blueprints for soft references before generating schema so that they are loaded into memory. |
-| Level streaming is not supported without a significant performance degradation. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but at the cost of poorer performance. |
+| Level streaming is not supported without a significant performance degradation. | 2019-03-26 | [UNR-1163](https://improbableio.atlassian.net/browse/UNR-1163) | Select the **Query Based Interest Enabled** checkbox in the SpatialOS GDK for Unreal Runtime Settings to enable or disable level streaming. If the **Query Based Interest Enabled** checkbox is ticked, level streaming will be supported, but this might affect performance in certain scenarios and is currently being investigated. |
```

Comment:
Level streaming is not supported without a significant performance degradation => Level streaming might affect performance in certain scenarios